test/Crate/Test/DBAL/Functional/TableOptionsTest.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * Licensed to CRATE Technology GmbH("Crate") under one or more contributor
+ * license agreements.  See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.  Crate licenses
+ * this file to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial agreement.
+ */
+
+namespace Crate\Test\DBAL\Functional;
+
+
+use Crate\DBAL\Platforms\CratePlatform;
+use Crate\Test\DBAL\DBALFunctionalTestCase;
+use Doctrine\DBAL\Schema\Table;
+
+class TableOptionsTest extends DBALFunctionalTestCase {
+
+    public function testAdditionalTableOptions() {
+        $platform = $this->_conn->getDatabasePlatform();
+
+        $options = [];
+        $options['sharding_column'] = 'id';
```

Comment:
I would like to see a test that shows that identifiers are quoted if needed:
e.g.:

```suggestion
        $options['sharding_column'] = 'idColumn';
```